dist/images/ovnkube.sh

```diff
@@ -929,10 +930,17 @@ ovn-node() {
       "
   }
 
+  if test -z "${OVN_UNPRIVILEGED_MODE+x}" -o "x${OVN_UNPRIVILEGED_MODE}" = xoff; then
+    OVN_UNPRIVILEGED_MODE=""
+  else
+    OVN_UNPRIVILEGED_MODE="--unprivileged-mode"
+  fi
+
```

Comment:
@dyusupov OVN_UNPRIVILEGED_MODE environment variable is overloaded here to mean "on/off" and also to.mean the CLI option. For the CLI option case above, L933 to L937, I would use a new variable like we do for other feature flags, for example: see multicast_enabled_flag. So, maybe: unprivelged_mode_flag?

Reply:
good point will change!

Reply:
Girish, I like it much better now! Please take a look again ...